frontend/src/metabase/core/components/TimeInput/TimeInput.tsx

```diff
@@ -15,7 +15,7 @@ import {
 
 export interface TimeInputProps {
   value: Moment;
-  is24HourMode?: boolean;
+  timeFormat?: string;
```

Comment:
I wonder what format will be used when it's not explicitly set?
Should we maybe set a default now, so it's a bit more clear?

Reply:
Fixed